shell/platform/windows/flutter_windows_engine.h

```diff
@@ -254,6 +255,12 @@ class FlutterWindowsEngine {
   // Updates accessibility, e.g. switch to high contrast mode
   void UpdateAccessibilityFeatures(FlutterAccessibilityFeature flags);
 
+  // Called when the application quits in response to a quit request.
+  void OnQuit(int64_t exit_code);
```

Comment:
Is this the right type? 

Looking at [ExitProcess](https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-exitprocess?redirectedfrom=MSDN), exit codes are [UINT](https://learn.microsoft.com/en-us/windows/win32/winprog/windows-data-types#uint) which are an unsigned [INT](https://learn.microsoft.com/en-us/windows/win32/winprog/windows-data-types#int)
which claims to be a 32-bit integer.